aws/wafv2_helper.go

```diff
@@ -457,20 +568,116 @@ func expandWafv2RuleAction(l []interface{}) *wafv2.RuleAction {
 	action := &wafv2.RuleAction{}
 
 	if v, ok := m["allow"]; ok && len(v.([]interface{})) > 0 {
-		action.Allow = &wafv2.AllowAction{}
+		action.Allow = expandWafv2AllowAction(v.([]interface{}))
 	}
 
 	if v, ok := m["block"]; ok && len(v.([]interface{})) > 0 {
-		action.Block = &wafv2.BlockAction{}
+		action.Block = expandWafv2BlockAction(v.([]interface{}))
 	}
 
 	if v, ok := m["count"]; ok && len(v.([]interface{})) > 0 {
-		action.Count = &wafv2.CountAction{}
+		action.Count = expandWafv2CountAction(v.([]interface{}))
+	}
+
+	return action
+}
+
+func expandWafv2AllowAction(l []interface{}) *wafv2.AllowAction {
+	action := &wafv2.AllowAction{}
+	if len(l) == 0 || l[0] == nil {
+		return action
+	}
+
+	m := l[0].(map[string]interface{})
+	if v, ok := m["custom_request_handling"]; ok && len(v.([]interface{})) > 0 {
+		action.CustomRequestHandling = expandWafv2CustomRequestHandling(v.([]interface{}))
+	}
+
+	return action
+}
+
+func expandWafv2CountAction(l []interface{}) *wafv2.CountAction {
+	action := &wafv2.CountAction{}
+	if len(l) == 0 || l[0] == nil {
+		return action
+	}
+
+	m := l[0].(map[string]interface{})
+	if v, ok := m["custom_request_handling"]; ok && len(v.([]interface{})) > 0 {
+		action.CustomRequestHandling = expandWafv2CustomRequestHandling(v.([]interface{}))
```

Comment:
similar comment as above:

```suggestion
	if len(l) == 0 || l[0] == nil {
		return nil
	}

	m, ok := l[0].(map[string]interface{})
	if !ok {
	        return nil
	}
	
	action := &wafv2.CountAction{}
	
	if v, ok := m["custom_request_handling"].([]interface{}); ok && len(v) > 0 {
		action.CustomRequestHandling = expandWafv2CustomRequestHandling(v)
```